project/sites/cosicani/themes/cosicani_theme/templates/layout/page.html.twig

```diff
@@ -93,29 +93,55 @@
     </h1>
   {% endif %}
   {% endif %}
+  <div class="front-banner">
+    <img
+      src="sites/cosicani/themes/cosicani_theme/images/cosica-banner.png"
+      alt="">
+  </div>
   {{ page.front_welcome }}
 
   {% if page.top_first or page.top_second or page.top_third %}
     <div id="top-area" class="clearfix">
       {% if page.top_first %}
-        <div class="column">
+        <div id="home-block-1" class="column">
```

Comment:
Nothing wrong with doing this but you could reduce specificity by just using the column class
e.g. column:first-child, column:nth-child(2), column:last-child?